applications/CompressiblePotentialFlowApplication/custom_elements/transonic_perturbation_potential_flow_element.cpp

```diff
@@ -379,6 +411,21 @@ void TransonicPerturbationPotentialFlowElement<TDim, TNumNodes>::GetWakeDistance
     noalias(distances) = GetValue(WAKE_ELEMENTAL_DISTANCES);
 }
 
+template <int TDim, int TNumNodes>
+void TransonicPerturbationPotentialFlowElement<TDim, TNumNodes>::GetEquationIdVectorExtendedElement(
+    EquationIdVectorType& rResult) const
+{
+    const int additional_node_id = GetAdditionalNode();
+
+    const auto& r_geometry = this->GetGeometry();
+    for (unsigned int i = 0; i < TNumNodes; i++)
+    {
+        rResult[i] = r_geometry[i].GetDof(VELOCITY_POTENTIAL).EquationId();
+    }
```

Comment:
This is what is done in GetEquationIdVectorNormalElement so you could just call this function directly instead of reimplementing.